src/Translation/DataValueTranslator.php

```
@@ -89,6 +116,59 @@ public function translateDecimalValue( DecimalValue $value ): SMWDataItem {
 		return new \SMWDINumber( $value->getValueFloat() );
 	}
 
+	private function translateEDTF( String $value ): \SMWDITime {
+		wfDebug( 'swb: translate edtf' );
+		$tvb = new TimeValueBuilder( EdtfFactory::newParser() );
+		$tvArr = $tvb->edtfToTimeValues( $value );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$tvArr'. (undefined)

(UnusedLocalVariable)